src/opendr/engine/target.py

```diff
@@ -437,11 +437,13 @@ class BoundingBoxList(Target):
     """
     def __init__(
         self,
-        boxes,
+        boxes=[],
```

Comment:
Default value shouldn't be a mutable value, i would suggest defaulting to `None` and inside the method check for `None` and assign empty list if so.

Reply:
Fixed